package/doc/sphinx/source/documentation_pages/lib/augment.rst

```diff
@@ -0,0 +1,2 @@
+.. automodule:: MDAnalysis.lib.augment

```

Comment:
the doc build failed because you refer to `lib.augment` here but you named the module `lib._augment` elsewhere